fineract-provider/src/integrationTest/java/org/apache/fineract/integrationtests/GroupSavingsIntegrationTest.java

```diff
@@ -160,10 +160,10 @@ public void testSavingsAccount_CLOSE_APPLICATION() {
 
         DateFormat dateFormat = new SimpleDateFormat("dd MMMM yyyy", Locale.US);
         Calendar todaysDate = Calendar.getInstance();
-        final String CLOSEDON_DATE = dateFormat.format(todaysDate.getTime());
+        final String closedOnDateValue = dateFormat.format(todaysDate.getTime());
         String withdrawBalance = "false";
```

Comment:
Is there a reason for adding ...Value for this specifically?

Reply:
updated.